### PR TITLE
Update CHANGELOG for API v3

### PR DESCRIPTION
**Pull request by fauna-ryanharris**
### Notes
Updates the `CHANGELOG.md` for the changes made in the release of `v3.0.1`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 3.0.0
+
+- Contains function has been renamed to ContainsPath
```
Comment (nathaniel-may) [inline CHANGELOG.md:3]: We should mention that `Contains` was deprecated in favor of `ContainsPath`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+- ContainsField, ContainsValue and Reverse functions added
```
Comment (BrunoQuaresma) [inline CHANGELOG.md:4]: I would say to rephrase to "Rename Contains function to ContainsPath" and "Add ContainsField, ContainsValue and Reverse functions".

Comment (faunaee) [reply CHANGELOG.md:4]: The documentation uses this wording:

```
* Added an alias of the Contains function called ContainsPath, and deprecated the Contains function.

* Added the new functions ContainsField and ContainsValue functions, to make it easier to explore the structure of objects and documents.
```

File: CHANGELOG.md (hunk 1 above)
Comment (fauna-ryanharris) [reply CHANGELOG.md:3]: @nathaniel-may Aren't we waiting until API v4 to deprecate that function?

Comment (faunaee) [reply CHANGELOG.md:3]: No.

Comment (faunaee) [reply CHANGELOG.md:3]: Deprecate !== remove.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+## 3.0.0
+
+- Added an alias of the Contains function called ContainsPath, and deprecated the Contains function.
+- Added the new functions ContainsField and ContainsValue functions to make it easier to explore the structure of objects and documents.
+- Added the new Reverse function to reverse the order of items in an Array, Page or Set
```
Comment (faunaee) [inline CHANGELOG.md:5]:
```suggestion
- Added the new Reverse function to reverse the order of items in an Array, Page, or Set.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+- Add support for versioned `@query` values
```
Comment (faunaee) [inline CHANGELOG.md:6]:
```suggestion
- Add support for versioned `@query` values.
```

**Pull request merged**